include/vapor/SliceRenderer.h

```diff
@@ -14,7 +14,9 @@
 #include <vapor/utils.h>
 #include <vapor/Renderer.h>
 
-#define DEBUG 1
+// clang-format off
```

Comment:
Why is it necessary to disable clang-format in this relatively cut-n-dry header file ? We've not yet decided what, if anything, we plan to do with clang-format moving forward. We decided to discuss this issue at our post mortem after the release.

Reply:
Removed.  

To answer your question: 
At our meeting two weeks ago we were discussing clang-format.  I said I'd like to disable it for my work in this PR because it was impeding my work.  Nobody objected to it, and it's used on a subjective basis elsewhere in the repo.

Reply:
I remember the conversation, but maybe I misunderstood. I thought there was some particular, isolated problem that you were running into, not planning on wholesale disabling the formatter.